apluslms_roman/observer.py

```diff
@@ -31,14 +35,17 @@ def enter_build(self):
         self._message(self._state, Message.ENTER)
     def done(self, data=None):
         self._state = Phase.DONE
-        self._message(self._state, Message.ENTER, data=data)
+        self._message(self._state, Message.BUILD_END, data=data)
 
     def start_step(self, step):
         self._message(self._state, Message.START_STEP, step)
 
     def end_step(self, step):
         self._message(self._state, Message.END_STEP, step)
 
+    def error(self, error_code):
```

Comment:
Let's merge `feature/new_observer` first and then rewrite color stuff on top of that. It has more feature complete observer interface.